flaky gluon consan test

### Describe the bug

repro, disable all but `test_clc_result_visibility` in `python/test/gluon/test_consan.py`:

GB200:
```
$ pytest python/test/gluon/test_consan.py --tb=short -v -n16 --flake-finder
...
================================ short test summary info ================================
FAILED python/test/gluon/test_consan.py::test_clc_result_visibility[39-4ctas-True] - AssertionError: assert 'Buffer being accessed has outstanding writes' in '/home/scra...
FAILED python/test/gluon/test_consan.py::test_clc_result_visibility[48-4ctas-True] - AssertionError: assert 'Buffer being accessed has outstanding writes' in '/home/scra...
======================= 2 failed, 298 passed in 335.60s (0:05:35) =======================
```

B300:
```
$ pytest python/test/gluon/test_consan.py --tb=short -v -n16 --flake-finder
...
================================ short test summary info ================================
FAILED python/test/gluon/test_consan.py::test_clc_result_visibility[0-4ctas-True] - AssertionError: assert 'Buffer being accessed has outstanding writes' in '/home/scra...
FAILED python/test/gluon/test_consan.py::test_clc_result_visibility[15-4ctas-True] - AssertionError: assert 'Buffer being accessed has outstanding writes' in '/home/scra...
FAILED python/test/gluon/test_consan.py::test_clc_result_visibility[21-4ctas-True] - AssertionError: assert 'Buffer being accessed has outstanding writes' in '/home/scra...
FAILED python/test/gluon/test_consan.py::test_clc_result_visibility[17-4ctas-True] - AssertionError: assert 'Buffer being accessed has outstanding writes' in '/home/scra...
FAILED python/test/gluon/test_consan.py::test_clc_result_visibility[24-4ctas-True] - AssertionError: assert 'Buffer being accessed has outstanding writes' in '/home/scra...
FAILED python/test/gluon/test_consan.py::test_clc_result_visibility[40-4ctas-True] - AssertionError: assert 'Buffer being accessed has outstanding writes' in '/home/scra...
FAILED python/test/gluon/test_consan.py::test_clc_result_visibility[41-4ctas-True] - AssertionError: assert 'Buffer being accessed has outstanding writes' in '/home/scra...
FAILED python/test/gluon/test_consan.py::test_clc_result_visibility[45-4ctas-True] - AssertionError: assert 'Buffer being accessed has outstanding writes' in '/home/scra...
======================= 8 failed, 292 passed in 149.47s (0:02:29) =======================
```


### Environment details

commit: e0399390bbc6d5ebcc0d5d248d12d5af0c0c9d23
system: GB200 (arm) and B300 (x86_64)

--- a/python/test/gluon/test_consan.py
+++ b/python/test/gluon/test_consan.py
@@ -373,7 +373,10 @@ def test_clc_result_visibility(FAILURE, device, run_wrapper, monkeypatch, num_ct
         result = run_in_process(test_clc_result_visibility, (FAILURE, device, False, monkeypatch, num_ctas))
         if FAILURE:
             assert_expected_cuda_failure(result.exc)
-            assert "Buffer being accessed has outstanding writes" in result.driver_stderr_output
+            assert any(msg in result.driver_stderr_output for msg in [
+                "Buffer being accessed has outstanding writes",
+                "Buffer being accessed has outstanding reads",
+            ])
         else:
             assert result.exc is None
             assert result.driver_stderr_output == ""

